articles/connections/apple-siwa/add-siwa-to-native-app.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Add Sign In with Apple to Native iOS Apps
+description: Learn how to add native login functionality to your native app with Apple. 
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Native iOS Apps
+
+You can add functionality to your native iOS application that allows your users to use Sign In with Apple to authenticate. In addition, you can also try the Auth0 [iOS Swift - Sign In with Apple Quickstart](/quickstart/native/ios-swift-siwa).
+
+## How it works
+
+For a native app, the Sign in with Apple login flow works as follows:
+
+![Sign In with Apple Authentication Flow](/media/articles/connections/social/apple/apple-siwa-authn-flow.png)
```

Comment:
Some notes on this diagram:

- Step #3: only the auth code (no tokens) are sent from the app to Auth0

Reply:
This is for Sonny - will resolve on our call.